modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/display/context/FDSAdminItemSelectorDisplayContext.java

```diff
@@ -19,8 +21,13 @@
 public class FDSAdminItemSelectorDisplayContext {
 
 	public FDSAdminItemSelectorDisplayContext(
-		HttpServletRequest httpServletRequest) {
+		HttpServletRequest httpServletRequest,
+		String itemSelectedEventName,
+		InfoItemItemSelectorCriterion infoItemItemSelectorCriterion) {
 
+		_classPK = ParamUtil.getLong(httpServletRequest, "refererClassPK");
```

Comment:
The `itemSelector` sends a `refererClassPK` param through the [itemSelectorURL param](https://github.com/liferay-frontend/liferay-portal/pull/4362/files#diff-b094c5f25620284eeb36a69b9802d581e15de4acf2397664ca9b1ff1dd398542R70), but it is always set to `0`.